### ISSUE TITLE
the source of the CMU fbx files

### ISSUE THREAD

**Issue by yinkangning0124** [nv-tlabs/ASE#74]
Hi,
    I have successfully retarget the motion seq from your provided cmu fbx file to my own robot. However, when downloading other cmu seqs from the webisite(https://forum.unity.com/threads/huge-free-fbx-mocap-library-released.261401/) I failed to retarget it. The reason maybe that the tpose of cmu seq from the website if different from the ones you provided. So can you tell me where you got these cmu fbx files? Or did you do some post-processing for the original cmu fbx files?
    
 Thanks.

**Comment by Croolch** [comment]
Did you check the given cmu mocap fbx file? I imported it in blender and found there was a tpose at the first keyframe. 
I guess this is a post processing. Not sure about it.

**Comment by yinkangning0124** [comment]
> Did you check the given cmu mocap fbx file? I imported it in blender and found there was a tpose at the first keyframe. I guess this is a post processing. Not sure about it.

**Comment by yinkangning0124** [comment]
> Did you check the given cmu mocap fbx file? I imported it in blender and found there was a tpose at the first keyframe. I guess this is a post processing. Not sure about it.



> Did you check the given cmu mocap fbx file? I imported it in blender and found there was a tpose at the first keyframe. I guess this is a post processing. Not sure about it.

Yeah, the first frame is a tpose, you can start from the second frame

**Comment by Croolch** [comment]
Hi, 

I have tried to retarget cmu mocap to amp humanoid mjcf model, and it worked well. 

I got the mocap from repo [cmubvh](https://github.com/Shriinivas/cmubvh). I used blender to transform bvh file to fbx file, and then using poselib scripts (fbx_importer, retarget_motion) from this repository to generate cmu npy file and retarget to humanoid. There is no other manipulation used.

Maybe you can try on this mocap data?

**Comment by yinkangning0124** [comment]
> Hi,
> 
> I have tried to retarget cmu mocap to amp humanoid mjcf model, and it worked well.
> 
> I got the mocap from repo [cmubvh](https://github.com/Shriinivas/cmubvh). I used blender to transform bvh file to fbx file, and then using poselib scripts (fbx_importer, retarget_motion) from this repository to generate cmu npy file and retarget to humanoid. There is no other manipulation used.
> 
> Maybe you can try on this mocap data?
Cool, I will try this. Thanks a lot !
